### PR TITLE
Add no renewal email error message

### PR DESCRIPTION
**Pull request by sorinmarta**
closes https://github.com/Strategy11/business-directory-premium/issues/224

This PR adds a new error message if the user doesn't have any recurring email templates when they press the `Send renewal email` button.

### CODE REVIEW

File: includes/admin/class-admin.php [hunk 1]
```diff
@@ -1075,6 +1075,21 @@ function handle_actions() {
 						break;
 					}
 
+					$all_notices      = wpbdp_get_option( 'expiration-notices' );
```
Comment (stephywells) [inline includes/admin/class-admin.php:1078]: Can we move this new code into a separate function to avoid adding extra complexity?

Comment (sorinmarta) [reply includes/admin/class-admin.php]: Good catch! Done, thanks!

**Review by stephywells — APPROVED** [review]
Thank you! This code looks great. Can we ask Garret to test it before merging?

**Review by garretlaxton — APPROVED** [review]
Sweet, this works great!

**Pull request merged**